ntoskrnl/mm/ARM3/section.c

```diff
@@ -3648,9 +3656,10 @@ NtMapViewOfSection(IN HANDLE SectionHandle,
 #endif
 
     /* Check for invalid inherit disposition */
-    if ((InheritDisposition > ViewUnmap) || (InheritDisposition < ViewShare))
+    if ((InheritDisposition < ViewShare) || (InheritDisposition > ViewUnmap))
     {
-        DPRINT1("Invalid inherit disposition\n");
+        DPRINT1("Invalid inherit disposition: %d\n", InheritDisposition);
+        // For local use only: ASSERT((InheritDisposition >= ViewShare) && (InheritDisposition <= ViewUnmap));
```

Comment:
You may remove that comment in the final version of this PR.

Reply:
My point is to have this `ASSERT` available to whoever wants to enable it.

Reply:
Since it's already inside if condition, it could be simply ASSERT(FALSE);

Reply:
We can all read code.

Reply:
I think the comments can be safely removed now. With that I'm OK with the PR. I mean that was what this PR was about, right?

Reply:
Removed commented out `ASSERT()`.